homeassistant/components/tolo/strings.json

```diff
@@ -61,6 +61,13 @@
           "automatic": "Automatic",
           "manual": "Manual"
         }
+      },
+      "aroma_therapy_slot": {
+        "name": "Aroma therapy slot",
+        "state": {
+          "a": "A",
+          "b": "B"
```

Comment:
What does this mean?

Reply:
The sauna offers to select between two slots, referred to as "A" and "B" (in the Android app, paper documentation of the hardware, etc.). So for this I want to reflect the values "A" and "B" as well in HA. However, HA requires translation strings to be lowercase, so here I "translate" it back to uppercase.

Any suggestion how to improve that?

Reply:
Does Too also show this in their app or is it like "Slot A" and "Slot B"?

Reply:
Actually, it's showing just "A" and "B", but I'm also open to change it here to "Slot A"/"Slot B".

![image](https://github.com/home-assistant/core/assets/781926/4a5f0ced-7df5-404d-adc2-66a15e7a9281)

Reply:
Hmm, I dont have a strong preference, I'll leave that up to you. Let me know what you think and I can merge this

Reply:
I think for the sake of readbility even "Slot A/B" might be better. Just pushed the update. Thanks for raising this in first place.